src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -100,8 +137,6 @@ export class HTMLMarkdownConverter {
 				const notebookLink = node.href ? URI.parse(node.href) : URI.file(node.title);
 				const notebookFolder = this.notebookUri ? path.join(path.dirname(this.notebookUri.fsPath), path.sep) : '';
 				let relativePath = findPathRelativeToContent(notebookFolder, notebookLink);
-				node.innerText = escapeAngleBrackets(node.innerText);
-				content = escapeAngleBrackets(content);
```

Comment:
Why is this escaping being removed?

Reply:
It wasn't working properly (not handling nested elements correctly). So, if I had the following:

```
<mark>He**llo** world</mark>
```

The unhighlight result would be
```Hello world```

Where instead, it should be

```He**llo** world```

Instead, now leveraging the text escaping mechanism.